pkg/apis/k8s.cni.cncf.io/v1/types.go

```diff
@@ -47,6 +47,7 @@ type NetworkStatus struct {
 	Mac       string   `json:"mac,omitempty"`
 	Default   bool     `json:"default,omitempty"`
 	DNS       DNS      `json:"dns,omitempty"`
+	DeviceID  string   `json:"deviceID,omitempty"`
```

Comment:
Do we need to make `deviceID` a map in case multiple devices in single net-attach-def? like the issue you mentioned in network-resources-injector https://github.com/intel/network-resources-injector/issues/18 or in case of a chained plugin: https://github.com/intel/multus-cni/pull/448

Reply:
In both cases, there is only one `deviceID `per network. isn't it ? 
For the issue https://github.com/intel/network-resources-injector/issues/18, another resource is OVS bridge which is required only at worker node and can be hidden away from container application. 
If we still have any compelling use case which need distinct resources per network, then we can change `DeviceID `into a `map `?

@JanScheurich, Please let us know your feedback.

Reply:
I am not aware of any type of secondary network attachment that would be associated with more than one PCI device ID.
The Userspace CNI also produces some device identification that must be available to containers for consumption of the virtio-user or memif devices. As far as I understand they stores this in shared files today. Perhaps an extensible container in the Network-Status pod annotation would be an alternative for them.